python/practices/practice-3/src/path1/age_condition_checker.py

```diff
@@ -10,37 +10,54 @@
 
 def check_condition_number(number):
     """
-    This function prints a number if it adds 36 to its inverse
+    This function returns a number if it adds 36 to its inverse
 
     Parameters:
         - number (string): a string of number to check condition.
+
+    Returns:
+        - string: the input number if it satisfies the condition,
+        otherwise None.
     """
 
     list_of_digits = list(number)
     number_reverse = int(''.join(list_of_digits[::-1]))
 
     if int(number) + REVERSE_ADDITION_NUMBER == number_reverse:
-        print(number)
+        return number
+
+    else:
+        return None
```

Comment:
This block of code needs to be refactored

Reply:
Done

Reply:
@hothanhtuan1209 you misunderstood. I would like to use only one `return` statement...
instead of
```
if int(number) + REVERSE_ADDITION_NUMBER == number_reverse:   
         return number
     else:
         return None
```

Reply:
Done